### PR TITLE
Chore/149448565/write documentation

### PR DESCRIPTION
**Pull request by codejockie**
#149448565 write api documentation

    #### What does this PR do?
    #### This PR adds documentation

    #### Description of Tasks to be completed
    #### N/A

    #### How should this be manually tested?
    #### N/A

    #### Any background context you want to provide?
    #### No

    #### What are the relevant pivotal tracker stories?
    #### Write API Documentation

    #### Screenshots (if appropriate)
    #### None

    #### Questions:
    #### None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [![npm](https://img.shields.io/npm/l/express.svg)](https://github.com/codejockie/document-manager)
+[![Build Status](https://travis-ci.org/codejockie/document-manager.svg?branch=master)](https://travis-ci.org/codejockie/document-manager)
 [![Code Climate](https://codeclimate.com/github/codejockie/document-manager/badges/gpa.svg)](https://codeclimate.com/github/codejockie/document-manager)
+[![Coverage Status](https://coveralls.io/repos/github/codejockie/document-manager/badge.svg?branch=master)](https://coveralls.io/github/codejockie/document-manager?branch=master)
 [![Test Coverage](https://codeclimate.com/github/codejockie/document-manager/badges/coverage.svg)](https://codeclimate.com/github/codejockie/document-manager/coverage)
```
Comment (BolajiOlajide) [inline README.md]: What's the essence of having two coverage badges in your `README`?

Comment (BolajiOlajide) [inline README.md]: Any reason in particular?

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+  - secure: XyRi4wHLXcjZdJ2vA+NKTAyZLvWxIu1OyjxDzhAM/fx2vQLVcFwV9s1d6H+J8JBokL/0//5UsCjzyjbeoZbxn/rLlCdAG4EeS7o4Ua4WFlssFUTaLvsuJQ0nF/RlOM7ztEkSroM5521jdnxqmli6GPBZexcqJCAIK+3GT0CxcP2w03ziQYsMa0F6qoEidzEVNYduK7a43yrlV2Ik56OEOPEYq/PBFl7pfwUR0w7iykn1Diy66HxYRqYGXko/ePE4p0J4bLzkE/wKqN6FtXcsRoGEG3x7S+uBvmPHBBd/WpMDTr9czMUL4a5CDReev+oJLzJyOvnl1qVU0XOOnkLB8GCYG4nPYxv7zvOs/Zktu4Hdh971oWvxxXozzjNz5dmQOSyM/Fp8Knjhzk80dbd/ytTnGThQuzmXVtO+VJm+snFw8HVIf2ZG0cQoUa6bh6PuBf9Njnzit410k0sdkwsXsdiO7DbACiF4deQ89kAGaYR6X6xpCuSciEbK4f0GrTAYIqatsNFcWTgA9eL5vYEPIcoHtILsZcaNfEeeKnk0qmDk6nO1BHZDAw2xlrKWzionNNJ2Skgi+1cGHE12CnX+ovJ8+P8Mr9ipxch355rWaSKh7uZBMFFJ4sn8reZVrpnMtzw4QVthZKkIJ6IoZ4wTWMi334wsNTgE2+hr2RsUq84=
```
Comment (BolajiOlajide) [inline .travis.yml]: This is a secret and as such should not be here. You don't even need this much in your `.travis.yml` file

File: README.md [hunk 3]
```diff
@@ -1,20 +1,112 @@
...
+- **[<code>GET</code> documents/:id]()** - Get a document
+- **[<code>POST</code> documents]()** - Create a document
+- **[<code>PUT</code> documents/:id]()** - Update a document
+- **[<code>DELETE</code> documents/:id/]()** - Delete a document
+
+#### User Resources
+
+- **[<code>GET</code> users]()** - Get all users
+- **[<code>GET</code> users/:id]()** - Get a user
+- **[<code>GET</code> users/:id/documents]()** - Get user's document
```
Comment (BolajiOlajide) [inline README.md]: ## Nitpick

`Users` not `User's`

File: .travis.yml (hunk 2 above)
Comment (codejockie) [inline .travis.yml]: Actually it's an encrypted secret that can only be interpreted by Travis CI

File: README.md (hunk 1 above)
Comment (codejockie) [inline README.md]: No reason, just a way of getting familiar with using both.

File: tests/routes/documents.spec.js [hunk 4]
```diff
@@ -10,9 +10,9 @@ const Role = require('../../build/models/index').Role;
 const request = supertest.agent(app);
 chai.use(chaiHttp);
 
-const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJrZW5uZWR5Lm53YW9yZ3VAYW5kZWxhLmNvbSIsInVzZXJuYW1lIjoiY29kZWpvY2tpZSIsImlhdCI6MTUwMDY2MTAwMywiZXhwIjoxNTAwOTIwMjAzfQ.8x36eu9hgEGkrWCYcH2ImA2z7N7OXtzOazVRM0GlaEA';
+const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJrZW5uZWR5Lm53YW9yZ3VAYW5kZWxhLmNvbSIsInVzZXJuYW1lIjoiY29kZWpvY2tpZSIsImlhdCI6MTUwMDcyODYxMCwiZXhwIjoxNTMyMjY0NjEwfQ.mn1ns_DZzP7vZKYR7_0EWm-8-HF_jk59jCQCmI0V6WY';
```
Comment (BolajiOlajide) [inline tests/routes/documents.spec.js]: All these tokens can be hidden in your `.env` file.

File: tests/routes/documents.spec.js [hunk 5]
```diff
@@ -10,9 +10,9 @@ const Role = require('../../build/models/index').Role;
 const request = supertest.agent(app);
 chai.use(chaiHttp);
 
-const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJrZW5uZWR5Lm53YW9yZ3VAYW5kZWxhLmNvbSIsInVzZXJuYW1lIjoiY29kZWpvY2tpZSIsImlhdCI6MTUwMDcyODYxMCwiZXhwIjoxNTMyMjY0NjEwfQ.mn1ns_DZzP7vZKYR7_0EWm-8-HF_jk59jCQCmI0V6WY';
+const authToken = process.env.AUTH_TOKEN;
```
Comment (BolajiOlajide) [inline tests/routes/documents.spec.js]: Good!

File: tests/routes/documents.spec.js (hunk 4 above)
Comment (codejockie) [inline tests/routes/documents.spec.js]: I've done that, in a new branch, yet to push it.

**Pull request merged**